libopae/plugins/xfpga/metrics/metrics_max10.c

```diff
@@ -59,6 +59,14 @@
 #include "metrics/metrics_metadata.h"
 #include "metrics/max10_metadata.h"
 
+// Max10 Metric limits
+#define THERMAL_HIGH_LIMIT             300.00
+#define THERMAL_LOW_LIMIT              -273.00
+#define POWER_HIGH_LIMIT               1000.00
+#define POWER_LOW_LIMIT                0.00
+#define VOLTAMP_HIGH_LIMIT             500.00
+#define VOLTAMP_LOW_LIMIT              0.00
```

Comment:
These limits are true for all MAX10, regardless of the platform?

Reply:
yes
Max10 /BMC limits